Cargo.toml

```diff
@@ -125,28 +125,28 @@ members = [
     "storage/backup/backup-service",
     "storage/inspector",
     "storage/jellyfish-merkle",
-    "storage/libradb",
-    "storage/librarian",
+    "storage/diemdb",
+    "storage/diemrian",
```

Comment:
We probably want to think a bit more about some of these

Reply:
`librarian`, lulz

Reply:
Good catch... I'm going through and making sure I didn't fat-finger anything else...